refarch-backend/src/main/resources/application.yml

```diff
@@ -1,6 +1,9 @@
 spring:
   application:
     name: @project.artifactId@
+  flyway:
+    locations:
+    - classpath:db/migration/schema
```

Comment:
@eidottermihi This looks interesting and useful, did you take a look at this?

Reply:
da hat der coderabbit etwas fantasiert, denn

clean-disbled: default ist bereits true
validate-on-migrate: default ist bereits true

zu baseline-on-migrate (https://documentation.red-gate.com/fd/baseline-on-migrate-224919695.html):

> This is useful for initial Flyway production deployments on projects with an existing DB.
>
> Be careful when enabling this as it removes the safety net that ensures Flyway does not migrate the wrong database in case of a configuration mistake!

Hat also nur entfernt was mit "Allows Flyway to work with existing databases" zu tun und macht in unserem Use Case keinen Sinn.

Ich würde also alles so lassen wie es ist: 
![3dvurz](https://github.com/user-attachments/assets/d29cb58f-d549-4e0b-ae59-1fa3a76fbbcc)